tfhub_dev/assets/sayakpaul/cartoongan/dr/1.md

```diff
@@ -0,0 +1,28 @@
+# Lite sayakpaul/east-text-detector/dr/1
```

Comment:
This handle is already used in https://raw.githubusercontent.com/tensorflow/hub/master/tfhub_dev/assets/sayakpaul/east-text-detector/dr/1.md

Is it possible to choose a different handle?

Reply:
Fixed it up.